docs/source/intro.rst

```diff
@@ -5,29 +5,28 @@ Getting Started
 ************
 Installation
 ************
-Installing rubato is simple. Just run the following in a terminal:
+Installing rubato is simple. Make sure you have Python 3.10 or later installed, then run the following in a terminal:
 
 .. code-block:: console
 
-    $ pip install rubato
+    $ pip3 install rubato
 
 .. note::
-    | Python 3.10.x required
-    | We recommend installing python direct from python.org if you do not already have it.
+    | We recommend installing python directly from `python.org <https://www.python.org/downloads/>`_ if you do not already have it.
 
 On Windows and Mac, the dll files that SDL needs come pre-bundled with rubato, so you should be ready to go.
 
 .. dropdown:: On Linux **only**, you'll need to install them yourself:
 
     .. code-block:: console
 
-        $ pip install pysdl2-dll
+        $ pip3 install pysdl2-dll
```

Comment:
if you want to be explicit, then we should do `python -m pip install pysdl2-dll`. That's the standard

Reply:
also specifying to use python3 is bad practice *i think*

Reply:
pip3 comes bundled with python 3.x and should work fine. it's better than just pip because on mac pip sometimes defaults to using python 2.7. I see no need to do python -m pip though